airflow/contrib/operators/bigquery_to_gcs.py

```diff
@@ -43,7 +43,7 @@ class BigQueryToCloudStorageOperator(BaseOperator):
     :param compression: Type of compression to use.
     :type compression: string
     :param export_format: File format to export.
-    :type field_delimiter: string
+    :type export_format: string
     :param field_delimiter: The delimiter to use when extracting to a CSV.
     :type field_delimiter: string
```

Comment:
string -> str?

Reply:
Same comment: I am creating a follow PR to do this as this (string -> str) is going to affect 100s of files. :)